ccd-definition/ComplexTypes.json

```diff
@@ -1278,5 +1278,14 @@
     "FieldType": "TextArea",
     "ElementLabel": "Directions and interim orders, if needed",
     "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "StandardDirections",
+    "ListElementCode": "standardDirectionsDocument",
```

Comment:
Do not use ComplexType for something that has only one property.

Reply:
i used complex type for consistency, in the tab view it wouldnt be encapsulated within the complex type box and there was no steer on the design so i didnt know if it should look like the rest of the form with that

Reply:
I do get it however complex types are supposed to group related properties like classes.

Reply:
ok